src/devices/rojaflex.c

```diff
@@ -0,0 +1,317 @@
+/** @file
+    Rojaflex shutter and remote devices.
+
+    Copyright (c) 2021 Sebastian Hofmann <sebastian.hofmann+rtl433@posteo.de>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+ */
+
+#include "decoder.h"
+
+/**
+Frequency: 433.92MH
+.modulation  = FSK_PULSE_PCM,
+.short_width = 100,
+.long_width  = 100,
+.reset_limit = 102400,
+*/
+
+/**
+Signal documentation
+
+Default signal layout
+0xaaaaaaaa d391d391 SS KKKKKK ?CDDDD TTTT CCCC
+
+4 Bytes Preamble
+4 Bytes Sync Word
```

Comment:
Can you add it as a comment? I assume it is d391d391.

Reply:
done